pkg/openshift/templates/push-ups.json.tpl.go

```diff
@@ -0,0 +1,146 @@
+package templates
+
+//PushUPSTemplate defines the template for deploying a push via UPS environment service to openshift 3
+var PushUPSTemplate = `
+{{define "push-ups"}}
+{
+  	"kind": "Template",
+  	"apiVersion": "v1",
+  	"metadata": {
+    	"name": "RHMAP UPS",
+    	"annotations": {
+      		"description": "RHMAP ups",
+			"dependencies": "data-mysql",
+      		"tags": "rhmap,mobile,nodejs",
+      		"iconClass": "icon-jboss"
+    	}
+  	},
+  	"objects": [
+    	{
+			"kind": "Service",
+			"apiVersion": "v1",
+			"metadata": {
+				"name": "{{.ServiceName}}",
+				"labels": {
+					"name": "{{.ServiceName}}",
+					"rhmap/domain": "{{.Domain}}",
+					"rhmap/env": "{{.Env}}",
+					"rhmap/guid": "{{.CloudAppGUID}}",
```

Comment:
Having this label on everything has been causing some problems where we use the guid if it is present but most of the time it is an empty string. I think we should remove it from all but the cloud app. or else generate a unique value for it. Maybe create an issue for that. No need to change right now